recipe/meta.yaml

```diff
@@ -1,21 +1,21 @@
 {% set name = "cramjam" %}
-{% set version = "2.6.2" %}
+{% set version = "2.7.0" %}
```

Comment:
The latest version is 2.7.0.post1 (not available on PyPi) but it has minimal changes https://github.com/milesgranger/pyrus-cramjam/compare/v2.7.0...v2.7.0.post1

Reply:
@skupr-anaconda we should only take what's on PyPI.